app/src/main/java/org/wikipedia/yearinreview/YearInReviewSlides.kt

```diff
@@ -318,57 +318,85 @@ class YearInReviewSlides(
         )
     }
 
-    private fun loggedInHighlightScreen(): HighlightsScreen {
-        return HighlightsScreen(
-            highlights = buildList {
-                if (yearInReviewModel.localTopVisitedArticles.isNotEmpty()) {
-                    val topVisitedArticles = yearInReviewModel.localTopVisitedArticles.take(3)
-                    add(
-                        HighlightItem(
-                            title = context.resources.getQuantityString(R.plurals.year_in_review_highlights_logged_in_most_read_article_title, topVisitedArticles.size),
-                            items = topVisitedArticles,
-                            highlightColor = ComposeColors.Blue600
-                        )
+    private fun loggedInHighlightScreen(): HighlightsScreen? {
+        val highlights = buildList {
+            // Top visited articles
+            if (yearInReviewModel.localTopVisitedArticles.isNotEmpty()) {
+                val topVisitedArticles = yearInReviewModel.localTopVisitedArticles.take(3)
+                add(
+                    HighlightItem(
+                        title = context.resources.getQuantityString(R.plurals.year_in_review_highlights_logged_in_most_read_article_title, topVisitedArticles.size),
+                        items = topVisitedArticles,
+                        highlightColor = ComposeColors.Blue600
                     )
-                }
+                )
+            }
+
+            val hasMinimumReadingActivity = yearInReviewModel.localReadingArticlesCount >= YearInReviewViewModel.MIN_READING_ARTICLES ||
+                    yearInReviewModel.totalReadingTimeMinutes >= YearInReviewViewModel.MIN_READING_MINUTES
+
+            // Reading time
+            if (hasMinimumReadingActivity && yearInReviewModel.totalReadingTimeMinutes > 0) {
```

Comment:
Not sure, but this does not look quite right - you have `hasMinimumReadingActivity` with the `yearInReviewModel.totalReadingTimeMinutes >= YearInReviewViewModel.MIN_READING_MINUTES`, but also you want it to `yearInReviewModel.totalReadingTimeMinutes > 0`. 

Can we just do `yearInReviewModel.totalReadingTimeMinutes >= YearInReviewViewModel.MIN_READING_MINUTES`?